content-model/events.json

```diff
@@ -52,7 +52,16 @@
     "featuredImage" : {
       "type" : "Image",
       "config" : {
-        "label" : "Featured image"
+        "label" : "Featured image",
+        "thumbnails" : [ {
+          "name" : "thinVideo",
```

Comment:
My reasoning on this name is the `16:9` is video ([it's the standard for HD, Digital TV etc](https://en.wikipedia.org/wiki/16:9)).